lib/view/dialog/confirm_logout_dialog.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
```

Comment:
The only functionality the confirm logout dialog should have is letting the caller know if yes or no was pressed with Navigator.pop(context, true) or Navigator.pop(context, false) depending on the button pressed. The navigation should be taken care of by the page view.